fusillade-api.yml

```diff
@@ -4,164 +4,465 @@ info:
   description: |
     Human Cell Atlas Data Coordination Platform Fusillade app demo
 
-    Log in: [Auth0](/login?state=ABC&redirect_uri=https://auth.dev.data.humancellatlas.org/echo)
+    Log in: [Auth0](/login?state=ABC&redirect_uri=https://auth.data.humancellatlas.org/echo)
   version: "0.0.1"
-host: $API_DOMAIN_NAME
-schemes:
-  - https
-produces:
-  - application/json
-components:
-  securitySchemes:
-    Authorization:
-      type: http
-      scheme: bearer
-      in: header
-      bearerFormat: JWT
-#      type: apiKey
-#      name: Authorization
-#      in: header
-#    openId:
-#      type: openIdConnect
-#      openIdConnectUrl: https://accounts.google.com/.well-known/openid-configuration
-#    oauth2:
-#      type: oauth2
-#      flow:
-#        authorizationCode:
-#          authorizationUrl: https://accounts.google.com/o/oauth2/v2/auth
-#          scopes:
-#            https://www.googleapis.com/auth/userinfo.email: read user email
-# FIXME: https://github.com/swagger-api/swagger-ui/issues/3641
+servers:
+  - url: https://$API_DOMAIN_NAME
 paths:
   /login:
     get:
       summary: Send the user agent to an identity provider selector.
       parameters:
         - name: openid_provider
           in: query
+          schema:
+           type: object
         - name: nav_src
           in: query
+          schema:
+            type: string
+      responses:
+        200:
+          description: OK
+        default:
+          description: Unexpected error
   /policies/evaluate:
```

Comment:
Is this one missing:
```security:
        - bearerAuth: []
```
on purpose?

Reply:
I was on the fence about this

Reply:
I'll leave it to your judgement then.  It doesn't seem to be critical to have auth on that endpoint to me, but maybe a more experienced opinion here would be better.